ios/RNSentry.m

```diff
@@ -273,7 +274,9 @@ - (void)setEventEnvironmentTag:(SentryEvent *)event
     if ([NSJSONSerialization isValidJSONObject:envelopeDict]) {
         SentrySdkInfo *sdkInfo = [[SentrySdkInfo alloc] initWithDict:envelopeDict[@"header"]];
         SentryId *eventId = [[SentryId alloc] initWithUUIDString:envelopeDict[@"header"][@"event_id"]];
-        SentryEnvelopeHeader *envelopeHeader = [[SentryEnvelopeHeader alloc] initWithId:eventId sdkInfo:sdkInfo traceContext:nil];
+        // forward the baggage header, right now only trace_id and public_key
```

Comment:
Wait for https://github.com/getsentry/sentry-javascript/pull/5292 to be released.
@Lms24 please ping me once this is released :) thanks.

Reply:
Will do (will finish the PR today, so probably merged tomorrow and released soon afterwards)

Reply:
@Lms24 so `7.3.0` has all the changes required by https://github.com/getsentry/develop/pull/613 right?

Reply:
@marandaneto almost all of them (which is the reason I was hesitant to tag you). The only thing the SDK is currently still missing is the correct treatment of 3rd party baggage header items on incoming requests (we still propagate them to outgoing requests which we shouldn't). Currrently working on that and we'll release a path the next days.

Reply:
Thanks, we'll wait for this patch then.

Reply:
Thanks.